subgraphs/_reference_/src/sdk/util/numbers.ts

```diff
@@ -0,0 +1,46 @@
+import { BigDecimal, BigInt } from "@graphprotocol/graph-ts";
```

Comment:
If we are to consolidate code between lending, dex and bridge, this file is a candidate for consolidation too.

Reply:
Yes! This `util` folder is meant to be shared by all protocol types